proxy/src/config.rs

```diff
@@ -49,10 +60,19 @@ pub fn configure_tls(key_path: &str, cert_path: &str) -> anyhow::Result<TlsConfi
         keys.pop().map(rustls::PrivateKey).unwrap()
     };
 
+    let cert_chain_bytes = {
+        let mut cert_file = std::fs::File::open(cert_path)
+            .context(format!("Failed to open TLS cert file at '{cert_path}'."))?;
+        let mut cert_chain_bytes = Vec::new();
+        let _ = cert_file.read_to_end(&mut cert_chain_bytes)?;
+        cert_chain_bytes
+    };
```

Comment:
This used to be `let cert_chain_bytes = std::fs::read(cert_path).context("TLS cert file")?;`